django_project/core/settings/project.py

```diff
@@ -57,8 +58,8 @@
 # For third party libs like bootstrap should go in contrib.py
 PIPELINE_CSS['project'] = {
     'source_filenames': (
+        'css/bootstrap.css',
```

Comment:
so there are two problems with this line and Irwan:

1) this is project.py so we should only have things in here that we ourselves have written
2) bootstrap.css should be coming from CDN so we should not be putting it in pipeline